src/bench/coin_selection.cpp

```diff
@@ -48,8 +48,10 @@ static void CoinSelection(benchmark::State& state)
         bool bnb_used;
         CoinEligibilityFilter filter_standard(1, 6, 0);
         CoinSelectionParams coin_selection_params(false, 34, 148, CFeeRate(0), 0);
-        bool success = wallet.SelectCoinsMinConf(1003 * COIN, filter_standard, vCoins, setCoinsRet, nValueRet, coin_selection_params, bnb_used)
-                       || wallet.SelectCoinsMinConf(1003 * COIN, filter_standard, vCoins, setCoinsRet, nValueRet, coin_selection_params, bnb_used);
+        bool success = wallet.SelectCoinsMinConf(1003 * COIN, filter_standard, vCoins, setCoinsRet, nValueRet, coin_selection_params, bnb_used);
+        if (!success) {
+            success = wallet.SelectCoinsMinConf(1003 * COIN, filter_standard, vCoins, setCoinsRet, nValueRet, coin_selection_params, bnb_used);
```

Comment:
Perhaps it should also be clarified why the two are expected to have different results given that they have exactly the same input parameters. I assume that the first pass sets bnb_used to true, but @achow101 should be able to clarify.

Reply:
This seems like a mistake. The first pass should be using BnB with its own CoinSelectionParams which signals for BnB to be used.

Reply:
@achow101 What is the correct `CoinSelectionParams` for the first pass? :-)

Reply:
```
CoinSelectionParams coin_selection_params(true, 34, 148, CFeeRate(0), 0);
```

Reply:
@achow101 Thanks! Please re-review updated version :-)